### DISCUSSION TITLE
Flagging buggy firmware

### DISCUSSION THREAD

**Discussion by alexforencich** [fwupd/fwupd#4841] [Q&A]
Is there any support in LVFS/fwupd for flagging buggy firmware versions?  If so, what is the appropriate channel to report this kind of issue?

The idea being that if there is a firmware version that is bricking devices, rendering systems unbootable, etc., then that version can be flagged so that fwupd does not attempt to install it, falling back to a non-flagged version (or at least raises major red flags that require manual intervention to override).  Potentially this could also be applied to releases with known major security issues, peripheral compatibility issues, etc.

**Comment by hughsie** [comment]
Yes, we have this issue (but thankfully *bricking* incidents are rare) and we have a few ways to co-ordinate this:

 * I get email about important issues, and I've demoted broken updates quite a few times. Broken even means "has no update descriptions" to me.
 * Most of the big vendors have issue trackers on the fwupd group that people report bugs to and I tag the correct firmware contacts depending on failure, for example. ME issues or missing firmware. e.g. see https://github.com/fwupd/firmware-dell
 * If an update fails to be deployed more 7% of the time (after we've seen more than 5 reports) we automatically demote it out of stable
 * You can do client-side ignoring of updates in `daemon.conf` if there's a specific version you want to avoid
